pkg/skaffold/yamltags/tags_test.go

```diff
@@ -277,8 +277,10 @@ func TestGetYamlTags(t *testing.T) {
 			expectedTags: []string{"c"},
 		},
 		{
-			name:         "non-pointer fields are returned in empty struct",
-			yaml:         struct{ A string }{},
+			name: "non-pointer fields are returned in empty struct",
+			yaml: struct {
+				A string `yaml:"a"`
```

Comment:
And some test cases with empty names too.

Reply:
done